tasks/html.js

```diff
@@ -6,42 +6,104 @@
  * Licensed under the MIT license.
  */
 
+var path = require('path');
+var chalk = require('chalk');
 var htmllint = require('../lib/htmllint');
 
 module.exports = function(grunt) {
   "use strict";
 
-  var chalk = require('chalk');
+  var usingDefaultReporter,
+
+    // Default Grunt reporter
+    defaultReporter = function(result) {
+      var out = result.map(function(message) {
+        var output = chalk.cyan(message.file) + ' ';
+        output += chalk.red('[') + chalk.yellow('L' + message.lastLine) +
+          chalk.red(':') + chalk.yellow('C' + message.lastColumn) + chalk.red('] ');
+        output += message.message;
+        return output;
+      });
+      return out.join('\n');
+    },
+
+    // Select a reporter (if not using the default Grunt reporter)
+    selectReporter = function(options) {
+      if (options.reporter === 'checkstyle') {
+        // Checkstyle XML reporter
+        options.reporter = '../lib/reporters/checkstyle.js';
+      } else if (options.reporter === 'json') {
+        // JSON reporter
+        options.reporter = '../lib/reporters/json.js';
+      } else if (options.reporter != null) {
+        // Custom reporter
+        options.reporter = path.resolve(process.cwd(), options.reporter);
+      }
+
+      var reporter;
+      if (options.reporter) {
+        try {
+          reporter = require(options.reporter).reporter;
```

Comment:
This `reporter` property isn't necessary, might as well drop that and change the two reporter implementations to assign to `module.exports` directly.